frontend/public/i18n.js

```diff
@@ -32,44 +32,17 @@ i18n
     detection: { caches: [] },
     contextSeparator: '~',
     ns: [
-      'add-users-modal',
       'badge',
-      'bindings',
-      'build',
-      'build-strategy',
       'ceph-storage-plugin',
```

Comment:
Hi, why are we removing namespaces outside of E-I?

Reply:
I did not remove these files in the a-d so just did it here.